pyam/core.py

```diff
@@ -405,6 +405,28 @@ def validate(self, criteria={}, exclude=False):
 
             return df
 
+    def conv_unit(self, unit_conv, inplace=False):
```

Comment:
why not use `convert_unit()` - still short and concise, but also immediately obvious?

Reply:
Agreed